content/reference/admin/private_locations/configuration/gcp/index.md

```diff
@@ -83,3 +94,35 @@ control-plane {
   ]
 }
 ```
+
+## Cloud NAT
+
+Cloud NAT (network address translation) lets certain resources in Google Cloud create outbound connections to the internet or to other Virtual Private Cloud (VPC) networks. 
```

Comment:
```suggestion
Cloud NAT (Network Address Translation) lets certain resources in Google Cloud create outbound connections to the Internet or to an other Virtual Private Cloud (VPC) networks. 
```

Reply:
Idk for the end of the sentence, it was copied from google documentation 🤔